src/screen/status.rs

```diff
@@ -249,6 +254,31 @@ fn create_status_section_items<'a>(
     .chain(items::create_diff_items(config, diff, &1, true))
 }
 
+fn create_stash_list_section_items<'a>(
```

Comment:
It'd be nice if this header didn't show when there are no stashed changes!

Reply:
Yeah, this makes sense, I will try to do that

Reply:
Done